packages/app/src/app/pages/Sandbox/Editor/Workspace/Project/SandboxConfig/Icon/index.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { useState, useEffect } from 'react';
```

Comment:
Remove unused import

```suggestion
import React, { useState } from 'react';
```